Frontend/src/components/photos/TutorialCarousel.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-container fluid >
+    <v-carousel height="90%" v-model="currentIndex">
+      <v-carousel-item v-for="(item,i) in items" :key="i" :src="item.src"></v-carousel-item>
```

Comment:
Besser:` <v-carousel-item
          v-for="item in items"
          :key="item.src"
          :src="item.src"/>`